### ISSUE TITLE
Test and deploy website to www.calconnect.org!

### ISSUE THREAD

**Issue by ronaldtse** [CalConnect/calconnect.org#50]
This website is ready in terms of content.

1. We need to test the website pages to ensure we have sufficient content (there are some links that does not resolve even in the current site).
2. Then we can deploy the site to calconnect.org.

Over to you, @douglm !

**Comment by ronaldtse** [comment]
Documentation:
* https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site

**Comment by douglm** [comment]
As we had lost access to the cpanel for dreamhost I was unable to get a copy of the DNS records for our domains.
I think I've managed to reconstruct the, and I've switched the nameservers a few minutes ago. Currently it's still pointing at the old calconnect.org site - my first concern is that we don't lose email access. 

@ronaldtse I should set you up an aws id for the calconnect account and give you access to route53

**Comment by ronaldtse** [comment]
I already have an IAM account with the CalConnect AWS account, so all I need would be the privileges to work with route 53. Thanks!

**Comment by douglm** [comment]
If I change the A record for calconnect.org will we get a github.io cert?

**Comment by douglm** [comment]
I think you have full admin access to the aws account.

**Comment by ronaldtse** [comment]
@douglm our www.metanorma.org is hosted on GitHub Pages. It uses a SAN (Subject Alternative Name) certificate from GitHub.

**Comment by kwkwan** [comment]
In order to setup Github Pages for `www.calconnect.org`, please create `CNAME` record of `www.calconnect.org` to `calconnect.github.io`.

Then, update the Pages settings (https://github.com/CalConnect/calconnect.org/settings/pages) to set the custom domain to `www.calconnect.org`.

If you also want to set the root apex to Github, you should setup A records of `calconnect.org` to the following IPs:
```
185.199.108.153
185.199.109.153
185.199.110.153
185.199.111.153
```

**Comment by douglm** [comment]
OK - I made the switch but it doesn't look correct - missing styles?

**Comment by ronaldtse** [comment]
Thank you @douglm ! This is all fixed in #51
